package-lock.json

```diff
@@ -0,0 +1,10009 @@
+{
```

Comment:
This file should not be checked-in. Please run `git rm --cached package-lock.json` and commit.